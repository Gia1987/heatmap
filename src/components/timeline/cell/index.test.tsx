import { render } from "@testing-library/react";
import ontoData from "../../../assets/data/onto-transactions.json";
import dayjs from "dayjs";
import Cell from "./";

const Component = (
  props = { startDate: dayjs("2019-01-01"), index: 1, data: ontoData }
) => {
  return <Cell {...props} />;
};

describe("<Cell/>", () => {
  it("render rgb(235, 115, 120)", () => {
    const { container } = render(Component());

    expect(container).toMatchSnapshot();
  });
});
