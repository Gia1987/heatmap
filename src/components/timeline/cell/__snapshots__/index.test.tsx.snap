// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cell/> render rgb(235, 115, 120) 1`] = `
<div>
  <div
    class="timeline-cells-cell"
    style="background-color: rgb(235, 115, 120);"
  />
</div>
`;
